plugins/inputs/windows_event_log/wineventlog/wineventlogrecord.go

```diff
@@ -78,3 +82,47 @@ func (record *windowsEventLogRecord) Value() (valueString string, err error) {
 func (record *windowsEventLogRecord) Timestamp() string {
 	return fmt.Sprint(record.System.TimeCreated.SystemTime.UnixNano())
 }
+
+type EventData struct {
+	Values []Data `xml:",any"`
+}
+
+type UserData struct {
+	Values []Data `xml:",any"`
+}
+
+// UserData has slightly different schema than EventData so that we need to overrid this
+// unmarshal function to get similar structure
+//
+// https://learn.microsoft.com/en-us/windows/win32/wes/eventschema-userdatatype-complextype
+// https://learn.microsoft.com/en-us/windows/win32/wes/eventschema-eventdatatype-complextype
+func (u *UserData) UnmarshalXML(d *xml.Decoder, start xml.StartElement) error {
+	in := struct {
```

Comment:
Unmarshal it into `EventData`.

Reply:
oh...this makes sense. Will fix it in next commit